mmdet/datasets/coco_with_text.py

```diff
@@ -63,9 +63,12 @@ def _parse_ann_info(self, img_info, ann_info):
             if ann.get('ignore', False):
                 continue
             x1, y1, w, h = ann['bbox']
-            text = ann['text']['transcription'] if ann['text']['legible'] else ''
+            for text_key in ('text', 'attributes'):
+                if text_key in ann:
```

Comment:
Is there a chance that neither of those keys are present in the `ann` dict?

Reply:
there is always some chance that user pass invalid annotation. If you think that some check/assertion needed, I can add. But anyway `KeyError` will be raised next line in case of absence both of keys.